### ISSUE TITLE
Customize bootstrap file

### ISSUE THREAD

**Issue by giova333** [quarkusio/quarkus#8362]
Greetings.
I'm trying to build quarkus lambda function with SSL support.
I am referring to this doc file https://github.com/quarkusio/quarkus/blob/master/docs/src/main/asciidoc/amazon-lambda.adoc

I am trying to modify `function.zip` by creating zip.native under the `src/main` directory and add cacerts and libsunec.so and custom `bootstrap` file there. But at the end standard function.zip is produced without my custom file. Should I add some additional extension?

**Comment by pschyma** [comment]
The documentation seems incomplete. You need to change the assembly to reflect your modifications. See #5968.

**Comment by giova333** [comment]
Yes, thanks, make sense. This docs just claims that everything should work out of the box
and developers just need to follow certain folder structure and naming conventions. 

_There are times you may want to set a specific system properties or other arguments when lambda invokes your native quarkus lambda deployment. If you include a bootstrap script file within zip.native, the quarkus extension will automatically rename the executable to runner within function.zip and set the unix mode of the bootstrap script to executable._

**Comment by oztimpower** [comment]
This should work, it does for me. I use it on many projects.

What needs to be updated in the documentation or fixed in the build process (AmazonLambaProcessor.java) for it to work for you?

**Comment by drissamri** [comment]
Not entirely belongs here, but since it's about customizing the build, is there any way to change the zip name (function.zip)? CloudFormation doesn't allow you to deploy two zips in one project if they both have the same name. (Two maven quarkus modules and one deployment for both)

**Comment by oztimpower** [comment]
Currently no, it is function.zip, the purpose being a quick start to get people started. 

Can I suggest you do the file move in a Maven or Gradle task, with the appropriate dependency? There are standard tasks to do this.
